### ISSUE TITLE
Import content from pre-2007 handcrafted site

### ISSUE THREAD

**Issue by coyotebush** [CPLUG/cplug.org-old#14]
Posts were stored in the `news` table in MySQL.

**Comment by coyotebush** [comment]
There are about a dozen posts from 2005 to 2007.

**Comment by icco** [comment]
Are you going to do this?

**Comment by coyotebush** [comment]
Yeah, I've had a DB dump around for a while, just need to glue together a query (or an insert-statement parser) and an HTML -> markdown converter.
